chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CriteriaAttributeReportController.java

```diff
@@ -0,0 +1,230 @@
+package gov.healthit.chpl.web.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import gov.healthit.chpl.report.ReportDataManager;
+import gov.healthit.chpl.report.criteriaattribute.CodeSetListingReport;
+import gov.healthit.chpl.report.criteriaattribute.CodeSetReport;
+import gov.healthit.chpl.report.criteriaattribute.ConformanceMethodListingReport;
+import gov.healthit.chpl.report.criteriaattribute.ConformanceMethodReport;
+import gov.healthit.chpl.report.criteriaattribute.FunctionalityTestedListingReport;
+import gov.healthit.chpl.report.criteriaattribute.FunctionalityTestedReport;
+import gov.healthit.chpl.report.criteriaattribute.OptionalStandardListingReport;
+import gov.healthit.chpl.report.criteriaattribute.OptionalStandardReport;
+import gov.healthit.chpl.report.criteriaattribute.PrivacyAndSecurityFrameworkListingReport;
+import gov.healthit.chpl.report.criteriaattribute.PrivacyAndSecurityFrameworkReport;
+import gov.healthit.chpl.report.criteriaattribute.StandardListingReport;
+import gov.healthit.chpl.report.criteriaattribute.StandardReport;
+import gov.healthit.chpl.report.criteriaattribute.SvapListingReport;
+import gov.healthit.chpl.report.criteriaattribute.SvapReport;
+import gov.healthit.chpl.report.criteriaattribute.TestDataListingReport;
+import gov.healthit.chpl.report.criteriaattribute.TestDataReport;
+import gov.healthit.chpl.report.criteriaattribute.TestToolListingReport;
+import gov.healthit.chpl.report.criteriaattribute.TestToolReport;
+import gov.healthit.chpl.util.SwaggerSecurityRequirement;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.extern.log4j.Log4j2;
+
+@Log4j2
+@Tag(name = "report-data/criteria-attributes", description = "Allows retrieval of data used by Criteria Attribute report.")
+@RestController
+@RequestMapping("/report-data/criteria-attributes")
+public class CriteriaAttributeReportController {
+    private ReportDataManager reportDataManager;
+
+    @Autowired
+    public CriteriaAttributeReportController(ReportDataManager reportDataManager) {
+        this.reportDataManager = reportDataManager;
+    }
+
+    @Operation(summary = "Retrieves the data used to generate the Test Tool Criteria Attribute Summary report.",
```

Comment:
Just want to note - I think the moving around of these endpoints might be a breaking change in the API. I don't care, I'm sure no one else used the endpoints, but just want to mention it since we didn't do the usual deprecation thing. Does the ticket need API-Impact?

Reply:
Could we leave the old endpoints in, as deprecated, and have them point to wherever these new endpoints go? At least then we're following our processes, and we can drop the old ones whenever we next to a major version update